### PR TITLE
fix(services): Set suggested services for testing results

### PR DESCRIPTION
**Pull request by Alchez**
Depends on https://github.com/DigiThinkIT/jhaudio_customizations/pull/1213.

### CODE REVIEW

**Review by DeveloperMetal — APPROVED** [review]

**Comment by DeveloperMetal** [comment]
;) What ever happened to having gifs on pr?

**Comment by Alchez** [comment]
Wait, this is still WIP. I need to sit down and discuss this with you :sweat_smile:

**Comment by DeveloperMetal** [comment]
ah, ic
wait why are we doing PRs for wip stuff?

**Comment by Alchez** [comment]
I really just put this up so you can have some reference to the code up for discussion.

**Comment by DeveloperMetal** [comment]
ah, cool
well, looks clean and straight forward so far lol
Just ping me if you want to continue the discussion whenever

File: repairs/public/js/warranty_claim.js [hunk 1]
```diff
@@ -132,18 +132,31 @@ frappe.ui.form.on("Warranty Claim", {
...
+									let driver_type = service_type == "driver" ? service_desc : "";
+
+									let suggested_service;
+									frappe.db.get_value("Item Service Default", {
+										"parent": frm.doc.item_code,
+										"service_type": service_type,
+										"driver_type": driver_type
+									}, "default_service_item", (r) => {
+										if (!r.exc) {
+											suggested_service = r.default_service_item;
```
Comment (Alchez) [inline repairs/public/js/warranty_claim.js]: @forellana-digithinkit 

I'm having a weird scope issue here. The variable that I've defined on line 144 is not being overriden here. I was planning to use it on line 159, but it's always being set as `null`.

Comment (DeveloperMetal) [reply repairs/public/js/warranty_claim.js]: so this is an async call. Where are you using suggested_service later?
If you are expecting to have this value right after calling frappe.db.get_value(...) you'll have a bad time

**Review by DeveloperMetal — APPROVED** [review]
Looks good, nice use of the async/await candy

**Pull request merged**